kalman_watch/options.py

```diff
@@ -0,0 +1,55 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Watch Kalman star data during perigee passages."""
```

Comment:
I ended up removing the use of astropy config, but keeping the config/conf pattern, since that can be changed and is not really that much more complicated that a plain module.